README.md

```diff
@@ -199,7 +199,7 @@ Then we can add an `npm test` script.
     "start": "nodemon lib/index.js --exec babel-node",
     "build": "babel lib -d dist",
     "serve": "node dist/index.js",
-+   "test": "mocha --compilers js:babel/register"
++   "test": "mocha --compilers js:babel-core/register"
```

Comment:
We still support `babel-core/register` but it will be deprecated: https://github.com/babel/babel/blob/master/packages/babel-core/register.js#L1